src/ngrx/4-create-actions/create-actions.md

```diff
@@ -23,7 +23,12 @@ Our goal is to have a different set of Action instead:
     - type: `[Login API] Login Failure`
     - props: `{ errorMsg: string }`
 - `logout` - dispatched when user clicks on a logout button
-    - type: `[Chat Page] Logout`
+    - type: `[Login Page] Logout`
```

Comment:
this should be dashboard page (look at text inside brackets as the source of the event)

Reply:
fixed!